spec/request/in_progress_forms_request_spec.rb

```diff
@@ -20,12 +21,32 @@
   end
 
   describe '#index' do
+    let(:user) { loa3_user }
```

Comment:
`:user` is set to be `loa1_user` in a other spec, letting `auth_header` switch between the two without any additional code.